config/graph/src/index.ts

```diff
@@ -227,6 +227,7 @@ export const MINICHEF_SUBGRAPH_NAME = {
   [ChainId.KAVA]: 'sushiswap/kava-minichef', //block subgraph not synced yet
   [ChainId.METIS]: 'sushiswap/metis-minichef',
   [ChainId.BOBA]: 'sushiswap/minichef-boba',
+  [ChainId.ARBITRUM_NOVA]: 'sushiswap/arbitrum-nova-minichef',
```

Comment:
I think it's called sushiswap/minichef-arbitrum-nova

Reply:
fixed

Reply:
oh yeah good call, it is minichef-arbitrum-nova